test/models_test.exs

```diff
@@ -53,12 +53,6 @@ defmodule Membrane.Telemetry.TimescaleDB.ModelTest do
       assert Enum.count(Repo.all(Measurement)) == 20
     end
 
-    test "returns error on duplicated Measurement" do
```

Comment:
Why this test is deleted?

Reply:
I came to a conclusion that we don't need (that it is not required) for the `time` column to bu unique, I deleted the uniqueness constraint as the after some time the index took more disk space than the measurements schema itself.